readthedocs/profiles/urls/public.py

```diff
@@ -7,7 +7,7 @@
 
 
 urlpatterns = [
-    url(r'^(?P<username>[\w@.-]+)/$',
+    url(r'^(?P<username>[+\w@.-]+)/$',
```

Comment:
I think the correct regex here is:

    [\w@\.\+-]+

which follows the same as the django validation field from the `username`.

Reply:
Forget about this comment, @stsewd is right here: https://github.com/rtfd/readthedocs.org/issues/3890#issuecomment-378441871

https://stackoverflow.com/questions/19976018/does-a-dot-have-to-be-escaped-in-a-character-class-square-brackets-of-a-regula

Reply:
@humitos no need to -1 yourself haha